### ISSUE TITLE
Accessibility: cant access links

### ISSUE THREAD

**Issue by shortwavesurfer2009** [hjiangsu/thunder#1825]
### Bug Description

When using the TalkBack at Screen Reader, I can see the link and I can focus on it, but I cannot click it.

### Expected Behaviour

I should be able to activate the link with a double tap. Just like you can with a single tap. If you do not have TalkBack turned on.

### Steps to Reproduce

1. Open post https://lemmy.dbzer0.com/post/42115357
2. Turn on talkback from android system settings
3. Flick from left to right with 1 finger until the archive.is link is read
4. Double tap

### Additional Context

https://github.com/user-attachments/assets/d0ce3464-0748-45b0-9457-02b82344e7b5

### App Version

0.7.0-1

### Device

OnePlus Nord N200 5G

### OS

Android 14

**Comment by micahmo** [comment]
Out of curiosity, do you think this behavior is related to #62 at all?

**Comment by shortwavesurfer2009** [comment]
yeah I think it does I'm almost certain it's got something to do with the thing you guys are using to display the text
